src/components/Contact_LangButton.js

```diff
@@ -0,0 +1,111 @@
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import HeadphonesOutlinedIcon from '@mui/icons-material/HeadphonesOutlined';
+import { Box, Button, MenuItem, Select, useMediaQuery } from '@mui/material';
+import { setCookie } from 'cookies-next';
+import { useTranslations } from 'next-intl';
+import { useRouter } from 'next/navigation';
+import { LOCALE_COOKIE } from '~/constants';
+import { useAppContext } from '~/contexts/AppContext';
+import theme from '~/theme';
+
+export default function Contact_LangButton({
+  languageOptions,
+  style,
+}) {
+  const t = useTranslations();
+  const router = useRouter();
+  const isMdUp = useMediaQuery(theme.breakpoints.up('md'));
+
+  const { language, setLanguage } = useAppContext();
+  const onChangeLanguage = (e) => {
+    const locale = e.target.value;
+    setCookie(LOCALE_COOKIE, locale);
+    setLanguage(locale);
+    router.refresh();
+  };
+
+  return (
+    <Box
+      sx={{
+        alignContent: 'center',
+        justifyContent: 'flex-end',
+        display: 'flex',
+        ...style,
+      }}
+    >
+      <Button
+        sx={{
+          color: 'black',
+          fontSize: {
+            xs: '10px',
+            md: '12px',
+          },
+          flexDirection: 'row',
+          flexWrap: 'wrap',
+          fontWeight: 'normal',
```

Comment:
redundant

Reply:
I checked this, it was the redundant thing.